string control_type
int32 number_actuators
